Signal/src/ViewControllers/ConversationView/ConversationViewItem.m

```diff
@@ -786,6 +786,11 @@ - (void)shareTextAction
         }
         case OWSMessageCellType_Unknown: {
             OWSFail(@"%@ No text to share", self.logTag);
+            break;
+        }
+        case OWSMessageCellType_ContactShare: {
+            OWSFail(@"%@ share contact not implemented.", self.logTag);
```

Comment:
I don't think we need these checks in production since `canSaveMedia` and `canPerformAction` should prevent us from ever reaching here.